cli/glnexus_cli.cc

```diff
@@ -550,15 +802,14 @@ GLnexus::Status load_config_preset(const std::string& name,
     return GLnexus::Status::OK();
 }
 
-
 void help_genotype(const char* prog) {
-    cerr << "usage: " << prog << " genotype [options] /db/path chrom:1234-2345" << endl
-         << "Genotype all samples in the database in the given interval. The positions are" << endl
+    cerr << "usage: " << prog << " genotype [options] /db/path /discovered_alleles/path chrom:1234-2345" << endl
```

Comment:
this subcommand no longer takes the chrom:1234-2345 positional argument, correct?